text-ui-test/ACTUAL.TXT

```diff
@@ -0,0 +1,47 @@
+Hello from
```

Comment:
ACTUAL.TXT file should not be in git :-p

Reply:
Thank you for the information Prof. We have stopped tracking ACTUAL.TXT in git.